src/components/FilterPicker/FilterPicker.tsx

```diff
@@ -45,7 +45,9 @@ export const FilterPicker = ({
                 filter.value === activeFilter.value
             )}
             id={`${filter.value}`}
-            label={filter.labelShort ?? filter.label}
+            label={filter.labelShort + ` ${[...Array(filter.value)]
```

Comment:
Hmm, maybe we should setup a function `levelToLabel` to avoid some repetition. I think we have this similar emoji logic in other places as well. What do you think? :)

Reply:
Also perhaps we can handle this outside filter picker since all filters are not of type impact or effort, checkout `getFilterPickerConfig`, this is where we configure the filter options

Reply:
> Hmm, maybe we should setup a function `levelToLabel` to avoid some repetition. I think we have this similar emoji logic in other places as well. What do you think? :)

Yeah, I think it could be nice to do something like that :) Should it be in it's own file, as `getFilterPickerConfig` is? Is that also what you meant here:

> Also perhaps we can handle this outside filter picker since all filters are not of type impact or effort, checkout `getFilterPickerConfig`, this is where we configure the filter options

Reply:
Yes I think it could be nice to have it in a separate file, perhaps in `src/utils`? Then we can call this method from both `getFilterPickerConfig` and `GuidelineCard`, does it make sense? :)